### PR TITLE
upgrade to substyle 3

### PR DESCRIPTION
**Pull request by jfschwarz**

### CODE REVIEW

**Comment by jfschwarz** [comment]
please review

File: package.json [hunk 1]
```diff
@@ -68,13 +68,14 @@
     "sinon": "~1.12.1",
     "sinon-chai": "^2.8.0",
     "style-loader": "^0.12.4",
+    "substyle": "^3.0.1",
```
Comment (frontendphil) [inline package.json]: Why is `substyle` included in the `dependencies` and `devDependencies`? Shouldn't one be enough?

Comment (jfschwarz) [inline package.json]: whoops, I have no idea how this slipped in there. good catch!

**Comment by jfschwarz** [comment]
> Why is this necessary actually?

It's an act of kindness to the portal guys who currently have to deal with a fucked up webpack setup which completely messes up node module resolving, resulting in modules getting wrong versions of their dependencies.

Anyways, nothing wrong with having all deps up-to-date...

**Comment by jfschwarz** [comment]
please complete your review, @frontendphil

**Pull request merged**